A_Bondar/HW2/HW1.1.py

```diff
@@ -0,0 +1,4 @@
+M = int(input('Введите рубли'))
+N = int(input('Введите копейки'))
+L = int(input('Введите количество товара'))
+print((M + N / 100) * L)
```

Comment:
with a price 1ruble 50 kopeks and 3 pieces your algorithm will give (50+1)*3/100=1.53

Reply:
I'm afraid, you are wrong. Please check the sequence of mathematical actions.
1. 50/100=0,5
2. 1+0,5=1,5
3. 1,5*3=4,5
Thanks.

Reply:
sure